src/commcare_cloud/fab/fabfile.py

```diff
@@ -387,7 +387,7 @@ def prepare_offline_deploy():
 def parse_int_or_exit(val):
     try:
         return int(val)
-    except ValueError:
+    except ValueError, TypeError:
```

Comment:
this should be expressed as an explicit tuple: `(ValueError, TypeError)`

(in python 2, `except Exception, e` is the same as `except Exception as e`)